### ISSUE TITLE
A11y: Gifs used in Labs are not a11y

### ISSUE THREAD

**Issue by daniellekirkwood** [vector-im/element-web#24961]
The picture marked in red is a gif, which is repeated again and again. There is no way to stop or pause the mapping.

The contents are therefore difficult to access for e.g. B. visually impaired users
who use magnification software and need more time to read. Additionally, users
with cognitive limitations, such as poor concentration, may be distracted from
capturing other content.

<img width="463" alt="Screenshot 2023-03-27 at 09 16 51" src="https://user-images.githubusercontent.com/89144281/227882456-eed7aaeb-1c2f-4dc0-9d3d-47e4fc5c697b.png">

**Comment by daniellekirkwood** [comment]
We don't currently have any Gifs in our Beta features (or Labs features) and therefore this can be closed. However, @nadonomy, how or where do we note that using Gifs in the Beta section of settings is not accessible? I think this can be led by the design team - do you have any ideas?

**Comment by nadonomy** [comment]
We discussed amongst the team, making a note to not use animated GIFs in future.
